package.json

```diff
@@ -39,12 +39,20 @@
         ]
       },
       {
-        "permissionName": "ui-remote-storage.settings.remote-storages.all",
-        "displayName": "Remote storage: Create, read, update, delete",
+        "permissionName": "ui-remote-storage.settings.remote-storages.view",
+        "displayName": "Remote storage: View",
         "visible": true,
         "subPermissions": [
           "remote-storage.configurations.collection.get",
           "remote-storage.configurations.item.get",
+          "settings.remote-storage.enabled"
+        ]
+      },
+      {
+        "permissionName": "ui-remote-storage.settings.remote-storages.edit",
+        "displayName": "Remote storage: Create, edit, delete",
+        "visible": true,
+        "subPermissions": [
           "remote-storage.configurations.item.post",
           "remote-storage.configurations.item.put",
           "remote-storage.configurations.item.delete",
```

Comment:
please add all the back-end permissions from
https://github.com/folio-org/mod-remote-storage/blob/de55fe971c0483f798777d8ab693b2d5cf0b2fe9/descriptors/ModuleDescriptor-template.json